runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewDelegate.java

```diff
@@ -161,8 +161,13 @@ public static void init(XWalkViewInternal xwalkView) throws UnsatisfiedLinkError
         }
 
         ResourceExtractor.setMandatoryPaksToExtract(MANDATORY_PAKS);
-        final int resourcesListResId = context.getResources().getIdentifier(
-                XWALK_RESOURCES_LIST_RES_NAME, "array", context.getPackageName());
+        int resListResId = context.getResources().getIdentifier(
+                XWALK_RESOURCES_LIST_RES_NAME, "array", context.getClass().getPackage().getName());
+        if (resListResId == 0) {
+            resListResId = context.getResources().getIdentifier(
```

Comment:
Here is a different reference to Cordova's approach: https://issues.apache.org/jira/browse/CB-5080
There they try getClass().getPackage().getName(), and if that does not work they try getPackageName(), which is the pattern I followed.